aggregate/aggregation_helpers.py

```diff
@@ -19,12 +19,12 @@ def order_aggregated_columns(
     df: pd.DataFrame,
     indicators_denom,
     categories,
-    household,
-    census_PUMS=False,
+    household=False,
+    exclude_denom=False,
```

Comment:
this would have bunch of downstream effects on the demographics work. Need to sort out depending which branch gets merged first.

Reply:
renaming the parameter in the function also auto-renames the parameter passed to the function because VScode re-names parameters across files. This definitely shouldn't break anything.

I also think setting a default for household is nice and shouldn't break any other accessors?

Reply:
yeah although for the work currently unavailable in the branch is not the case right? I think for the demographics stuff still in the PR probably might need an extra look at. But overall I feel like merging this PR from you would be easier first step.